pkgs/os-specific/linux/broadcom-sta/default.nix

```diff
@@ -66,7 +76,9 @@ stdenv.mkDerivation {
 
   nativeBuildInputs = kernel.moduleBuildDependencies;
 
-  patches = map (patch: "${rpmFusionPatches}/${patch}") patchset;
+  patches = map (patch: "${rpmFusionPatches}/${patch}") patchset ++ [
+    "${manjaro-patches}/017-linux612.patch"
```

Comment:
Why not use `fetchpatch2` with https://gitlab.manjaro.org/packages/extra/broadcom-wl-dkms/-/raw/9675acde490a9cbc5eb93aacafc1295e0658a77c/017-linux612.patch ?

Reply:
Good idea. I also changed the URL from Manjaro's repository to a primary source (a Gist posted by the patch author).